package.json

```diff
@@ -110,6 +110,7 @@
     "redux-thunk": "^2.1.0",
     "sass-loader": "^3.1.2",
     "style-loader": "^0.13.0",
+    "term.js": "rancher/term.js#583581949471d833849db71cc9b1d3b74d2d8474",
```

Comment:
We don't need all of `rancher/term.js`, just `https://raw.githubusercontent.com/rancher/term.js/master/src/term.js`.

Can we lock on to the specific revision+file?  Is it still broken? (I don't see any import fs business in src/term.js itself)